app/views/bids/index.html

```diff
@@ -0,0 +1,29 @@
+<h1>Bids for "<%= @auction.title %>"</h1>
+
+<a href="/auctions/<%= @auction.id %>">« Back to auction</a>
+
+<table class="usa-table-borderless">
+  <thead>
+    <tr>
+      <th scope="col">Bidder</th>
+      <th scope="col">Amount</th>
+      <th scope="col">Date (Eastern Standard Time)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @auction.bids.sort_by {|b| b.created_at}.reverse.each_with_index do |bid, i| %>
```

Comment:
I'll do that. And the feature spec should help catch and regressions (it tests the visual order of the table rows).